CollAction/Views/Projects/Details.cshtml

```diff
@@ -209,7 +212,51 @@
                 </div>
             </div>
         }
+    </div>
+    
+    @if (canTrack)
+    {
+        <div class="container">
```

Comment:
What is the use of this empty container?

Reply:
I added the <hr> in it to disqus_thread div column.